src/Compilers/Core/Desktop/CodeAnalysisDesktopResources.resx

```diff
@@ -132,4 +132,10 @@
   <data name="MissingKeepAlive" xml:space="preserve">
     <value>Missing argument for '/keepalive' option.</value>
   </data>
+  <data name="RuleSetBadAttributeValue" xml:space="preserve">
+    <value>The attribute {0} has an invalid value of {1}.</value>
+  </data>
+  <data name="RuleSetMissingAttribute" xml:space="preserve">
+    <value>The element {0} is missing an attribute named {1}.</value>
+  </data>
```

Comment:
To my knowledge there is no way to map an `XmlNode` back to a document location.  I double checked the API to make sure but didn't see anything that would help here.